src/snovault/elasticsearch/searches/mixins.py

```diff
@@ -56,6 +57,9 @@ def _get_facet_title(self, facet_name):
     def _get_facet_type(self, facet_name):
         return self._get_facets().get(facet_name, {}).get(TYPE_KEY, TERMS)
 
+    def _get_facet_open_on_load(self, facet_name):
```

Comment:
This looks okay but need to add test for `_get_facet_open_on_load` in `snovault/tests/test_searches_mixins.py`. 

Should be called`test_searches_mixins_aggs_to_facets_mixin_get_facet_open_on_load`.

Can see `test_searches_mixins_aggs_to_facets_mixin_get_facet_type` for example of how to mock response from `_get_facets`: https://github.com/ENCODE-DCC/snovault/blob/84c439a24587354903fff5977025795198ce1112/src/snovault/tests/test_searches_mixins.py#L1458